### ISSUE TITLE
Still Maintained?

### ISSUE THREAD

**Issue by Dan1jel** [ronilaukkarinen/weed#34]
Tried this but my irssi got destroyed. is this still maintaind? or Should it still work?

**Comment by ronilaukkarinen** [comment]
I hope you took a backup. This is inactive and not maintained but I'm happy to help. Can you elaborate on how it's broken? What problem it causes and in which step?

**Comment by Dan1jel** [comment]
> I hope you took a backup. This is inactive and not actively maintained but I'm happy to help. Can you elaborate on how it's broken? What problem it causes and in which step?

I sure did do a backup :) well i did was the setup explained but my Irssi did not look like it should and the bottom bar did not look as it would. I did restore it right after but i will try and redo it tomorrow. 

But i think it went crazy when i transfered all scripts and did /reload. 

But it would be nice if this could get reborn. Looks really good and i do think this is the best looking theme atm.

**Comment by ronilaukkarinen** [comment]
Yeah the instructions may not be so clear, it's been over 10 years actually when I did them for the first time... needs some rework but the theme is indeed still good, still using it daily with many others :)

As for the bottom bar, it's intented to be "empty" by default because the channel list is not really INSIDE the theme, it's actually created by adv_windowlist.pl script. Should be able to run it `/script load awl` (the version that comes with the theme), this is stated in the steps. You should also add autoruns to the scripts so that they will load when you start irssi.

**Comment by Dan1jel** [comment]
after searching around the web i found this [https://github.com/x70b1/irssi-weed](url) and it (what i state and what i think) is an updated verison. That one workes for me. 

Dont know really whats been updated tho. but atleast it works :) 

but is it possible to maybe just make the text hilighted when i get mention instead of a big red box taking up space in the chat? and maybe make the room text (green up top) to maybe white?  really like the idéa of the them otherwise.

know there is maybe questions thats not related to "maintained" but i ask it here instead of making a new "issue".
![weed](https://user-images.githubusercontent.com/4266424/109143742-a7c1e500-7760-11eb-871a-9e9f40bb27e8.png)

**Comment by ronilaukkarinen** [comment]
Fixed the incorrect repository url if you are wondering the edit I just made to your comment. Didn't realize that fork exists but after all there's 36 forks of my theme... and I get that because for example colors are mainly personal preferences and like all irssi themes this is meant to be hackable. If you don't like something, you are free to change it. In fact, now that you like to change the hihlight color, I suggest you make a fork to keep your version of the theme safe for you to use later if you happen to share the theme you have modified or just need to set up irssi all over again in some point of time in the future.

It's interesting that if @x70b1 has improvements in his fork he has not sent any Pull Requests for them... I would be glad to merge them if they are good (ping  @x70b1, let me know if you are interested in elaborating your changes more and the reasons why you separated the fork, always eager to hear them).

As for the highlight change, you can change it by modifying [this line](https://github.com/ronilaukkarinen/weed/blob/13c934392797a06810db82860f936f44b47b0fe0/weed.theme#L94):

```` perl
pubmsgmenick = "%1%W{msgnick $1-|}";
````

If you don't happen to knokw about irssi theming, let me describe this a bit for you. Colors are well documented [here](https://irssi.org/documentation/settings/#a_d). The red background is `%1` in the code above, so first take it off completely. The white text color is `%W`. If you want to change it to yellow **text** withou background for example, the line would be:

```` perl
pubmsgmenick = "%y{msgnick $1-|}";
````

As for the topic bar, I don't remember which it was but I guess you can fiddle with [this line](https://github.com/ronilaukkarinen/weed/blob/13c934392797a06810db82860f936f44b47b0fe0/weed.theme#L176).

I hope this helps!

**Comment by x70b1** [comment]
Hi guys!

Nice to see this development here.
It is a surprise that my fork is now getting attention.

I was looking for a cool irssi theme. It is quite clear that I have started to use `irssi-weed´. :sunglasses:

The idea of my fork was to change smaller stuff for my personal use case. But then I dived deeper. I have updated scripts and changed some more. While it got quieter in the main repository, people seem to like it. I even have stars on the repository already.

I have never opened PR's as there was also not much develeopment here. In #9 was also no movement.
And to be honest: For my personal se case it just worked how I did it. 

But if you want I could open PR's for the important things.

**Comment by ronilaukkarinen** [comment]
Hello! Thanks for the insight! I still use IRC and irssi daily, never stopped, it's been two decades already haha. However that particular issue #9 is something I don't do myself as I don't use much scripts any more (only advanced window list and highlightwindow to be honest because using mostly via [irssiproxy](https://github.com/ensc/irssi-proxy) (which is awesome btw, allows you to use ANY client and still retain irssi in the background in perfect sync [like this, all in harmony](https://twitter.com/rolle/status/1050263700060471296), look it up!) and [thelounge](https://github.com/thelounge/thelounge)) so I was expecting someone to chime in. As I'm busy with my day job as an front end developer I never noticed there is activity in someone's fork as no further feedback received. So this is a nice surprise for me as well.

To answer the original question of this ticket, yes, the repository is well maintained and in daily use for so many people.

That being said, please send improvements that you find would suit this repository well. I haven't even tested my methods for years since I have just wgetted the theme to instances I need and then just set up the rest by hand... sometimes the daily grind takes hold and you just try to manage. This is kinda what happened to this repo, it's still maintained as I said but kinda inactive during the years lately.

So, go for it! I'm going to close this issue right now as the questions are mainly answered. Well meant PRs always welcome.

**Comment by Dan1jel** [comment]
> Fixed the incorrect repository url if you are wondering the edit I just made to your comment. Didn't realize that fork exists but after all there's 36 forks of my theme... and I get that because for example colors are mainly personal preferences and like all irssi themes this is meant to be hackable. If you don't like something, you are free to change it. In fact, now that you like to change the hihlight color, I suggest you make a fork to keep your version of the theme safe for you to use later if you happen to share the theme you have modified or just need to set up irssi all over again in some point of time in the future.
> 
> It's interesting that if @x70b1 has improvements in his fork he has not sent any Pull Requests for them... I would be glad to merge them if they are good (ping  @x70b1, let me know if you are interested in elaborating your changes more and the reasons why you separated the fork, always eager to hear them).
> 
> As for the highlight change, you can change it by modifying [this line](https://github.com/ronilaukkarinen/weed/blob/13c934392797a06810db82860f936f44b47b0fe0/weed.theme#L94):
> 
> ```` perl
> pubmsgmenick = "%1%W{msgnick $1-|}";
> ````
> 
> If you don't happen to knokw about irssi theming, let me describe this a bit for you. Colors are well documented [here](https://irssi.org/documentation/settings/#a_d). The red background is `%1` in the code above, so first take it off completely. The white text color is `%W`. If you want to change it to yellow **text** without background for example, the line would be:
> 
> ```` perl
> pubmsgmenick = "%y{msgnick $1-|}";
> ````
> 
> As for the topic bar, I don't remember which it was but I guess you can fiddle with [this line](https://github.com/ronilaukkarinen/weed/blob/13c934392797a06810db82860f936f44b47b0fe0/weed.theme#L176).
> 
> I hope this helps!

Thanks for the quick and helpfull reply. I change a little at it looks great :) i hope i fins the green colour but i do like it better now as it is. I just searched "best Irssi theme" and this repo was number 3 from the top and it grew on me. After a little more search i found @x70b1 updated version and tried they one and it worked after first try. 

Still one of the best out there :) would like to see more Irssi themes but i think there are less and less people who use irssi now days.

**Comment by ronilaukkarinen** [comment]
@Dan1jel Thanks, that's awesome! I hope @x70b1 will send PRs to my way so we get this repo back in business as well. In fact I'm going to test it out soon and see what's changed over the years and what could be better.

I'm going to reopen this to try and keep this active.

**Comment by Dan1jel** [comment]
> @Dan1jel Thanks, that's awesome! I hope @x70b1 will send PRs to my way so we get this repo back in business as well. In fact I'm going to test it out soon and see what's changed over the years and what could be better.
> 
> I'm going to reopen this to try and keep this active.

That would be great, i can gladly test things out if you need help. I have me backup now so im at service if you need it!
